app/data/models/claim.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const claim = sequelize.define('claim', {
+    calculationId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    paymentRequestId: DataTypes.INTEGER,
+    sbi: DataTypes.INTEGER,
+    agreementNumber: DataTypes.STRING,
```

Comment:
Is this the same `agreementNumber` as given in `paymentRequests` table?

Reply:
Yes